script/1_0_2current_en_script/Arcueid/Day 4/04_00_ARC04_6_1.txt

```diff
@@ -7,44 +7,44 @@ I don't want to go through something like that ever again.
 If I stay with Arcueid, the same thing will definitely happen again. If I don't want that, I have no choice but to go back to the mansion.
 <Page3>
 "――――――"
-...There's a big flaw in my choice.
+...There's a big flaw in my choice.//Maybe revisit
 Knowing what it was and what I did wrong as a person, I keep a lid on my thoughts.
 I am not running away.
-I tell myself that I'm just returning back to normal.
+I tell myself that I'm just returning to the way things were.
 <Page4>
 "...I'll go back to the mansion. If I do that, then surely..."
-By the time dawn breaks, at a completely unrelated time, I will surely reach some sort of conclusion.
+By the time dawn breaks I'm sure that out there, in a place that's no longer any of my concern, some sort of conclusion will have been reached.
 <Page5>
-"Hey there. Can I have a word with you?"
+"Hey you. Do you have a second?"
 "Eh?"
 Just as I was about to get up from the bench, a voice called out to me from behind.
 <Page6>
 "Normally I'd leave you alone, but you have such a pathetic look on your face."
 <Page7>
 There is a girl standing there. I don't think I've ever seen her before, but she seems familiar.
-She seems to be a middle school student, but she is not wearing a uniform. I wonder if she is skipping school like I am.
+She seems to be a middle school student, but she isn't wearing a uniform. I wonder if she is skipping school like I am.
 <Page8>
 "....What exactly is pathetic?"
 "Your face. It has the same color as clay. That's the kind of face people have when they die, right?"
 <Page9>
 "――――――"
-That was very awkward, even for a joke.
-I couldn't laugh and respond to what she just said.
+Even as a joke, that's in poor taste.
+I can't laugh off what she just said.
 The girl is staring at me, not seeming to care that I have taken it so seriously.
 <Page10>
-"I'm not joking, I'm talking based on experience. A grandpa of someone I knew stopped breathing after three hours from looking like that. If I had called an ambulance, I might have been able to save him."
+"I'm not joking, I'm speaking based on experience. The grandpa of someone I know stopped breathing three hours after his face started looking like that. If I had called an ambulance, maybe he could have been saved."
 <Page11>
 The girl doesn't come close to me.
-She keeps her distance of about a meter,
+While keeping a distance of about a meter,
```

Comment:
I'm slightly inclined to keeping the original line but changing the comma to a full stop.

Reply:
Yeah I initially thought this one was a typo til I looked back at the script / the next page played. Changing to a full stop works for me.

Reply:
I agree with changing to a full stop, but the original is awkward and maybe grammatically incorrect. "She keeps her distance" is already a full phrase and adding the "of" to the end makes it... weird. It looks like it should be either "She keeps her distance" or "She keeps a distance of about a meter" as-is. I propose:
> She keeps her distance, staying about a meter away.